src/main/kotlin/com/bridgecrew/activities/PostStartupActivity.kt

```diff
@@ -1,23 +1,90 @@
 package com.bridgecrew.activities
-import com.bridgecrew.services.CheckovService
+import CheckovInstallerService
+import CliService
+import com.bridgecrew.listeners.CheckovInstallerListener
+import com.bridgecrew.listeners.CheckovScanListener
+import com.bridgecrew.listeners.CheckovSettingsListener
+import com.bridgecrew.services.CheckovScanService
+import com.bridgecrew.services.checkovRunner.CheckovRunner
+import com.bridgecrew.services.checkovRunner.PipCheckovRunner
+import com.bridgecrew.ui.CheckovToolWindowManagerPanel
+import com.bridgecrew.utils.PANELTYPE
+import com.bridgecrew.utils.getGitRepoName
+import com.intellij.openapi.application.ApplicationManager
 import com.intellij.openapi.startup.StartupActivity
 import com.intellij.openapi.project.Project
 import com.intellij.openapi.diagnostic.logger
 
 import com.intellij.openapi.components.service
+import java.nio.file.Paths
+
 private val LOG = logger<PostStartupActivity>()
 
 class PostStartupActivity : StartupActivity {
 
     override fun runActivity(project: Project) {
         LOG.info("Startup activity starting")
         this.installCheckovOnStartup(project)
+        PipCheckovRunner.getPythonUserBasePath(project)
+        getGitRepoName(project)
```

Comment:
why is it here? don't we only need this value for the checkov runs ?

Reply:
As the plugin cannot run as a sequence of codes anymore and now is event driven.
On startup we run async all the necessary data for the runs, that way when a run is activated we can get the result.